### PR TITLE
Revoke all full access roles

### PR DESCRIPTION
**Pull request by wszychta**

### CODE REVIEW

**Review by bmtoml — APPROVED** [review]
seems fine to me.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,8 @@
 ---
+# 1.5.1
+## Main Changes
+    - Revoke all unnecessary IAM permissions from EC2 instances - it was massive security issue
+
```
Comment (Maciej-Gierwatowski) [inline CHANGELOG.md:5]: I'd treat it as major change

File: main.tf [hunk 2]
```diff
@@ -15,14 +15,37 @@ resource "aws_ecs_cluster" "default" {
...
-    "arn:aws:iam::aws:policy/AmazonECS_FullAccess",
-    "arn:aws:iam::aws:policy/AWSApplicationDiscoveryServiceFullAccess",
-    "arn:aws:iam::aws:policy/service-role/AmazonEC2ContainerServiceforEC2Role",
-    "arn:aws:iam::aws:policy/AmazonRoute53FullAccess"
-  ]
+
+data "aws_iam_policy_document" "ecs_instance_policy" {
+  statement {
+    effect    = "Allow"
+    resources = ["*"]
```
Comment (Maciej-Gierwatowski) [inline main.tf:22]: Could we scale down privileges only to drone related resources?
Other resources should use  dedicated IAM keys

File: main.tf [hunk 3]
```diff
@@ -15,14 +15,37 @@ resource "aws_ecs_cluster" "default" {
...
+      "ecs:StartTelemetrySession",
+      "ecs:UpdateContainerInstancesState",
+      "ecs:Submit*",
+      "ecr:GetAuthorizationToken",
+      "ecr:BatchCheckLayerAvailability",
+      "ecr:GetDownloadUrlForLayer",
+      "ecr:BatchGetImage",
+      "logs:CreateLogStream",
+      "logs:PutLogEvents"
+    ]
```
Comment (Maciej-Gierwatowski) [inline main.tf:39]: Split statement to more specific, allowed actions per resource scope definitions. I.e. different statement for `ecr:*`, diferent `ecs:*` etc

File: CHANGELOG.md (hunk 1 above)
Comment (wszychta) [reply CHANGELOG.md:5]: upgraded to 1.6.0

File: main.tf (hunk 3 above)
Comment (wszychta) [reply main.tf:99]: done

File: main.tf (hunk 2 above)
Comment (wszychta) [reply main.tf:26]: done

**Review by Maciej-Gierwatowski — APPROVED** [review]

**Pull request merged**